bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -82,9 +82,27 @@ async function run() {
 				width: viewportWidth,
 			} );
 
-			console.log( `Resizing and saving to ${ file }` );
-			const image = await sharp( screenshot );
-			return await image.metadata().then( ( metadata ) => {
+			let image = await sharp( screenshot );
+
+			console.log( `Resizing and saving to ${ file }.webp` );
+			await image.metadata().then( ( metadata ) => {
+				image
+					.extract( {
+						// Ensure we're not extracting taller area than screenshot actaully is
+						height: Math.min( metadata.height, captureMaxHeight * viewportScaleFactor ),
+						left: 0,
+						top: 0,
+						width: metadata.width,
+					} )
+					.resize( outputWidth )
+					.webp() // default quality is 80
+					.toFile( `${ file }.webp` );
+			} );
+
+			image = await sharp( screenshot );
+			console.log( `Resizing and saving to ${ file }.jpg` );
+
+			await image.metadata().then( ( metadata ) => {
```

Comment:
I would do 
```js
[`${ file }.webp`, `${ file }.jpg`].forEach(file => 
...
```

And keep one copy of this logic to make the code DRYer.

Reply:
Great tip. Appreciate it!! 🙇